status/status_ext_test.go

```diff
@@ -20,9 +20,10 @@ package status_test
 
 import (
 	"errors"
+	"github.com/golang/protobuf/proto"
```

Comment:
Nit: Not sure why your editor is doing this. But we usually try to group imports into the following groups, separated by newlines: standard library imports, all other imports, generated pb.go imports, blank imports. So, it would be nice to keep the `proto` import out of the standard library group. Thanks. And same applies to other files in this PR as well.